### PR TITLE
cn-pr-blog-2016-04-00-Kubernetes-On-Aws_15

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

**Comment by chenrui333** [comment]
relates to https://github.com/kubernetes/website/pull/9469

File: content/cn/blog/_posts/2016-04-00-Kubernetes-On-Aws_15.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+<!-- ---
+title: " How to deploy secure, auditable, and reproducible Kubernetes clusters on AWS "
+date: 2016-04-15
+slug: kubernetes-on-aws_15
+url: /blog/2016/04/Kubernetes-On-Aws_15
+--- -->
+
+---
+title: " 如何在AWS上部署安全，可审计，可重现的k8s集群 "
+date: 2016-04-15
```
Comment (markthink) [inline content/cn/blog/_posts/2016-04-00-Kubernetes-On-Aws_15.md]: reproducible - `可重现` 建议翻译为：`可复现`

File: content/cn/blog/_posts/2016-04-00-Kubernetes-On-Aws_15.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+_Join us at [CoreOS Fest Berlin](https://coreos.com/fest/), the Open Source Distributed Systems Conference, and learn more about CoreOS and Kubernetes._ -->
+
+_今天的客座文章是由Colin Hom撰写，[CoreOS](https://coreos.com/)的基础架构工程师。CoreOS致力于推广谷歌的基础架构模式（Google’s Infrastructure for Everyone Else， #GIFEE），让全世界的容器都能在CoreOS Linux, Tectonic 和 Quay上安全运行。_
+
+_加入到我们的[柏林CoreOS盛宴](https://coreos.com/fest/)，这是一个开源分布式系统主题的会议，在这里可以了解到更多关于CoreOS和Kubernetes的信息。_
+
+<!-- At CoreOS, we're all about deploying Kubernetes in production at scale. Today we are excited to share a tool that makes deploying Kubernetes on Amazon Web Services (AWS) a breeze. Kube-aws is a tool for deploying auditable and reproducible Kubernetes clusters to AWS, currently used by CoreOS to spin up production clusters. -->
+
+在CoreOS, 我们一直都是在生产环境中大规模部署Kubernetes。今天我们非常兴奋地想分享一款工具，它能让你的Kubernetes生产环境大规模部署更加的轻松。Kube-aws这个工具可以用来在AWS上部署可审计，可重现的k8s集群，而CoreOS本身就在生产环境中使用它。
```
Comment (markthink) [inline content/cn/blog/_posts/2016-04-00-Kubernetes-On-Aws_15.md]: 可重现 => 可复现

File: content/cn/blog/_posts/2016-04-00-Kubernetes-On-Aws_15.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+<!-- Today you might be putting the Kubernetes components together in a more manual way. With this helpful tool, Kubernetes is delivered in a streamlined package to save time, minimize interdependencies and quickly create production-ready deployments. -->
+
+也许今天，你更多的可能是用手工的方式来拼接Kubernetes组件。但有了这个工具之后，Kubernetes可以流水化地打包、交付，节省时间，减少了相互间的依赖，更加快捷地实现生产环境的部署。
```
Comment (markthink) [inline content/cn/blog/_posts/2016-04-00-Kubernetes-On-Aws_15.md]: 建议中英文之间用空隔分开

**Review by markthink — COMMENTED** [review]
总体翻译非常棒，谢谢..

File: content/cn/blog/_posts/2016-04-00-Kubernetes-On-Aws_15.md (hunk 3 above)
Comment (chenrui333) [reply content/cn/blog/_posts/2016-04-00-Kubernetes-On-Aws_15.md]: 你的意思是 like this, `拼接Kubernetes组件` -> `拼接 Kubernetes 组件`

**Comment by chenrui333** [comment]
@markthink thanks for the review and encourage!

**Comment by chenrui333** [comment]
How to get this PR merged?

**Pull request merged**